src/components/molecules/ButtonFooter.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <div class="flex flex-col w-full items-center container mx-auto md:p-5 md:block">
+        <long-button colorClass="bg-btn-dark text-gray-light" text="Provide Feedback" />
```

Comment:
bg-btn-dark is not the same colour as in Figma. I just merge this branch to main and you can use the colour "primary-dark".